bundle/src/main/java/com/adobe/acs/commons/http/headers/impl/AbstractDispatcherCacheHeaderFilter.java

```diff
@@ -64,14 +65,14 @@
 
     /**
      * Get the value to place in the Cache-Control header.
-     * 
+     *
```

Comment:
Can you please cleanup the diff a bit? It's not only about the diff here in github, but also the standard git command line client does not have the "ignore whitespaces" setting as default.